packages/infra/lib/api-stack.ts

```diff
@@ -1192,7 +1192,7 @@ export class APIStack extends Stack {
 
     const writeIOPsMetric = dbCluster.metricVolumeWriteIOPs();
     const wIOPSAlarm = writeIOPsMetric.createAlarm(this, `${dbClusterName}VolumeWriteIOPsAlarm`, {
-      threshold: 10000, // IOPs per second
+      threshold: 100_000, // IOPs per second
```

Comment:
I think there's a typo here, its not only adding the separator but actually adding another zero (10x), right?